xml/System.Reflection/MetadataLoadContext.xml

```diff
@@ -60,9 +68,57 @@
         <ReturnType>System.Reflection.Assembly</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the assembly that denotes the "system assembly" that houses the well-known types such as <see cref="T:System.Int32" />.</summary>
+        <value>An <see cref="T:System.Reflection.Assembly" /> instance.</value>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+## Remarks
+
+The core assembly is treated differently than other assemblies because references to these well-known types do not include the assembly reference, unlike normal types.
+            
+Typically, this assembly is named "mscorlib", or "netstandard". If the core assembly cannot be found, the value will be null and many other reflection methods, including those that parse method signatures, will throw.
+            
+The <xref:System.Reflection.MetadataLoadContext.CoreAssembly> is determined by passing the coreAssemblyName parameter passed to the <xref:System.Reflection.MetadataAssemblyResolver> constructor
+to the MetadataAssemblyResolver's <xref:System.Reflection.MetadataAssemblyResolver.Resolve> method.
+
+If no coreAssemblyName argument was specified in the constructor of <xref:System.Reflection.MetadataLoadContext>, then default values are used including "mscorlib", "System.Runtime" and "netstandard".
+            
+The designated core assembly does not need to contain the core types directly. It can type forward them to other assemblies. Thus, it is perfectly permissible to use the mscorlib facade as the designated core assembly.
+            
+Note that <xref:System.Runtime> is not an ideal core assembly as it excludes some of the interop-related pseudo-custom attribute types such as DllImportAttribute. However, it can serve if you have no interest in those attributes. The CustomAttributes API will skip those attributes if the core assembly does not include the necessary types.
```

Comment:
```suggestion
Note that <xref:System.Runtime> is not an ideal core assembly because it excludes some of the interop-related pseudo-custom attribute types such as <xref:System.Runtime.InteropServices.DllImportAttribute>. However, it can serve if you have no interest in those attributes. The CustomAttributes API will skip those attributes if the core assembly does not include the necessary types.
```